bin/test/i18n/test_i18n_script_utils.rb

```diff
@@ -359,6 +359,28 @@ def around
     end
   end
 
+  describe '.write_json_file' do
+    let(:write_json_file) {I18nScriptUtils.write_json_file(file_path, data)}
+
+    let(:file_path) {'/expected/file.txt'}
+    let(:data) {{key: {key2: 'val'}}}
+
+    it 'writes pretty json content to the file' do
+      expected_file_content = <<-JSON.strip.gsub(/^ {8}/, '')
```

Comment:
I've seen this `strip. gsub` a few times now, is it supposed to remove 8 white spaces at the beginning?

Reply:
At the beginning of each new line

Reply:
At the beginning of each new line, but it's better to use `<<~` heredoc syntax instead